backend/danswer/llm/answering/answer.py

```diff
@@ -153,6 +156,7 @@ def __init__(
 
         self._return_contexts = return_contexts
         self.skip_gen_ai_answer_generation = skip_gen_ai_answer_generation
+        self._is_generating = True
```

Comment:
nit: prefer `is_cancelled` since that feels more clear as to what's happening. `is_generating` feels like a state of the internal rather than something that is triggered by the outside world

Reply:
Very fair - let's make the verbiage claer!